.xulieta.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <xulieta xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:noNamespaceSchemaLocation="./vendor/codelicia/xulieta/xulieta.xsd">
-    <plugin>Codelicia\XulietaJson\Plugin</plugin>
-    <plugin>Codelicia\XulietaJson\Plugin</plugin>
+    <validator>Codelicia\XulietaJson\JsonValidator</validator>
+    <validator>Codelicia\XulietaJson\JsonValidator</validator>
```

Comment:
Linking https://github.com/codelicia/xulieta/issues/59

Reply:
Should be fine to keep just one now.